src/services/dataset/DatasetService.js

```diff
@@ -1,57 +1,25 @@
 // Copyright (c) Cosmo Tech.
 // Licensed under the MIT license.
-
-import { CosmotechApiService } from '../../configs/Api.config';
-
-const DatasetApi = new CosmotechApiService.DatasetApi();
-
-function findAllDatasets (organizationId) {
-  return new Promise((resolve) => {
-    DatasetApi.findAllDatasets(organizationId, (error, data, response) => {
-      resolve({ error, data, response });
-    });
-  });
-}
+import { Api } from '../../configs/Api.config';
 
 function findDatasetById (organizationId, datasetId) {
```

Comment:
Do we really need the functions `createDataset`, `updateDataset` and `deleteDataset` ? If these functions are only wrappers around the call to the API, we should be consistent between all services and create these wrappers for all calls to the API

Reply:
By now, I let these "wrapper class" to not have a big breaking change between 1.3 and 1.4.
The ticket "Move to NPM Packages" (for v2) will normally fix this point.